radio/src/lua/api_general.cpp

```diff
@@ -875,6 +875,46 @@ static int luaDefaultStick(lua_State * L)
   return 1;
 }
 
+/* luadoc
+@function setTelemetryValue(name, id, subID, instance, value, unit, precision)
+
+@param name (string) Name of the sensor if it does not yet exit (4 chars)
+
+@param id Id of the sensor
+
+@param subID  subID of the sensor, usually 0
+
+@param instance instance of the sensor
+
+@param value of the sensor
+
+@param precision the precision of the sensor, value is devided by 10^precision, e.g. value=1000, prec=2 => 10.00
+
+@retval true, if the sensor was just added. In this case the value is ignored (subsequent call will set the value)
+*/
+static int luaSetTelemetryValue(lua_State * L)
+{
+  const char* name = luaL_checkstring(L, 1);
+  uint16_t id = luaL_checkinteger(L, 2);
+  uint8_t subId = luaL_checkinteger(L, 3);
+  uint8_t instance = luaL_checkinteger(L, 4);
+  int32_t value = luaL_checkinteger(L, 5);
+  uint32_t unit = luaL_checkinteger(L, 6);
+  uint32_t prec = luaL_checkinteger(L, 7);
```

Comment:
How about having  unit and prec as optional? Even the name could be optional. Typically, an unrecognized sensor takes up the id number as a name... What do you think?